Description: With automake 1.11.3, pkglib_PROGRAMS is not valid anymore, 
 so work around this by using my_execbin_PROGRAMS.
Author: Andreas Moog <amoog@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/934265

Upstream-Status: Backport 

Index: lcdproc-0.5.5/server/drivers/Makefile.am
===================================================================
--- lcdproc-0.5.5.orig/server/drivers/Makefile.am	2012-02-26 14:04:37.000000000 +0100
+++ lcdproc-0.5.5/server/drivers/Makefile.am	2012-02-26 14:13:17.764467350 +0100
@@ -20,7 +20,8 @@
 
 ## Keep the lists sorted!
 
-pkglib_PROGRAMS = @DRIVERS@
+my_execbindir = $(pkglibdir)
+my_execbin_PROGRAMS = @DRIVERS@
 EXTRA_PROGRAMS = bayrad CFontz CFontz633 CFontzPacket curses debug CwLnx ea65 EyeboxOne g15 glcdlib glk hd44780 icp_a106 imon imonlcd IOWarrior irman joy lb216 lcdm001 lcterm lirc lis MD8800 mdm166a ms6931 mtc_s16209x MtxOrb mx5000 NoritakeVFD picolcd pyramid sed1330 sed1520 serialPOS serialVFD shuttleVFD stv5730 SureElec svga t6963 text tyan sli ula200 xosd i2500vfd irtrans
 noinst_LIBRARIES = libLCD.a libbignum.a
 
